### PR TITLE
docs: Streamline csproj description texts for nuget

### PR DESCRIPTION
**Pull request by mfkl**

### CODE REVIEW

File: LibVLCSharp.WPF/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# LibVLCSharp.WPF
+
+[![NuGet Stats](https://img.shields.io/nuget/v/LibVLCSharp.WPF.svg)](https://www.nuget.org/packages/LibVLCSharp.WPF)
+[![NuGet Stats](https://img.shields.io/nuget/dt/LibVLCSharp.WPF.svg)](https://www.nuget.org/packages/LibVLCSharp.WPF)
+
+LibVLCSharp.WPF is the WPF integration for LibVLCSharp.
+
+It contains the views that allow to display a video played with [LibVLCSharp](../LibVLCSharp/README.md)
```
Comment (jeremyVignelles) [inline LibVLCSharp.WPF/README.md]: allows?

File: LibVLCSharp.WPF/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+in a WPF.
```
Comment (jeremyVignelles) [inline LibVLCSharp.WPF/README.md]: in a WPF app.

File: LibVLCSharp.WPF/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Windows
+
+## Why should I reference this package in my project?
+
+If you want to create a video application using WPF and C# with .NET, this package is made for you.
+
+You can also create a true cross-platform application with Xamarin.Forms, and use the WPF backend.
+In that case, you would need the [LibVLCSharp.Forms.Platforms.WPF](../LibVLCSharp.Forms.Platforms.WPF) package instead, which internally references this one.
+
+For other platforms, see the [main documentation](../README.md).
```
Comment (jeremyVignelles) [inline LibVLCSharp.WPF/README.md]: Can we add a few words about limitations like airspace?

File: LibVLCSharp.WinForms/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- net40+
+
+Supported platform:
+
+- Windows
+
+## Why should I reference this package in my project?
+
+If you want to create a video application using Windows Forms and C# with .NET, this package is made for you.
```
Comment (jeremyVignelles) [inline LibVLCSharp.WinForms/README.md]: Why do we limit to C#? It should work with VB.net as well, right?

File: LibVLCSharp/LibVLCSharp.csproj [hunk 5]
```diff
@@ -2,23 +2,30 @@
...
 
-      libvlc needs to be installed separately. See VideoLAN.LibVLC.* packages.
-    </Description>
+- Android
+- iOS
+- Mac
+- tvOS
+- UWP
+
+If you need Xamarin.Forms support, see LibVLCSharp.Forms. LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.</Description>
```
Comment (jeremyVignelles) [inline LibVLCSharp/LibVLCSharp.csproj]: Can you add a blank line before "LibVLC needs..."?

File: LibVLCSharp.WPF/README.md (hunk 1 above)
Comment (mfkl) [reply LibVLCSharp.WPF/README.md]: :D I actually wrote _allows_ first and then changed it. I'm not sure... It is the view**s** that allow, no?

Comment (jeremyVignelles) [reply LibVLCSharp.WPF/README.md]: You're right, I misread. 👍

**Pull request merged**